torchmdnet/module.py

```diff
@@ -73,6 +74,11 @@ def __init__(self, hparams, prior_model=None, mean=None, std=None):
         else:
             self.model = create_model(self.hparams, prior_model, mean, std)
 
+        self.ema_prmtrs = None
+        if self.hparams.ema_prmtrs_decay is not None:
```

Comment:
Tests fail because of this. You have to first make sure the element exists, see how its done with the charge above:
```python
        if "charge" not in hparams:
```
Alternatively you can deal with hparams as a dict before the call to save_hyperparameters.